presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -681,6 +681,11 @@ private PlanBuilder window(PlanBuilder subPlan, QuerySpecification node)
         return subPlan;
     }
 
+    private static List<SortItem> getSortItemsFromOrderBy(Optional<OrderBy> orderBy)
```

Comment:
I think this function could also have use in `StatementAnalyzer`. There are many places with similar usage